### PR TITLE
Migration script

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: src/scripts/migrateProjects.js [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+const projectService = require('@services/projects')
+const entityService = require('@services/entities')
+const resourceService = require('@services/resource')
+const resourceQueries = require('@database/queries/resources')
+const _ = require('lodash')
+const MongoClient = require('mongodb').MongoClient
+// var ObjectId = require('mongodb').ObjectID
+
+//get mongo db url
+const mongoUrl = process.env.MONGODB_URL
```
Comment (adithyadinesh0412) [inline src/scripts/migrateProjects.js]: can we add this in the `env.sample` also?
@priyanka-TL

File: src/scripts/migrateProjects.js [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+
+if (!mongoUrl) {
+	throw new Error('MONGODB_URL is not set in the environment variables.')
+}
+
+// Path to the CSV file
+const outputPath = path.resolve(__dirname, 'migration_results.csv')
```
Comment (adithyadinesh0412) [inline src/scripts/migrateProjects.js]: will the csv be able to handle if the result is volume is more? @priyanka-TL  
Should we go for a better approach?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
 	"dependencies": {
 		"@babel/parser": "^7.20.3",
-		"@babel/types": "^7.20.2"
+		"@babel/types": "^7.20.2",
+		"uuid": "^10.0.0"
```
Comment (aks30) [inline package.json]: @priyanka-TL why do we have 2 different package.json?

File: src/scripts/migrateProjects.js [hunk 4]
```diff
@@ -0,0 +1,472 @@
+/**
+ * name : migrateProjects.js
+ * author : Priyanka Pradeep
+ * created-date : 24-Aug-2024
+ * Description : script to create the project from consumption side.
+ */
+
```
Comment (aks30) [inline src/scripts/migrateProjects.js]: @priyanka-TL - Few questions?

We should use the org id of the creator for org id

Do we store the reference of mongo id in SCP?

Do we upload the JSON in cloud during migration?

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 14 Oct

File: src/scripts/migrateProjects.js (hunk 4 above)
Comment (priyanka-TL) [reply src/scripts/migrateProjects.js]: @aks30  Creator org Id is not available in template, Yes we are published_id. Yes uploading to cloud

File: package.json (hunk 3 above)
Comment (priyanka-TL) [reply package.json]: Removed

**Pull request merged**